employees/templates/employees/project_report_list.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% block content %}
+{% load rest_framework %}
+{% load static %}
+
+<link rel="stylesheet" type="text/css" href="{% static 'employees/style.css' %}">
+
+<h1>{{ project_name }}{{ UI_text.PAGE_TITLE.value }}</h1>
+<div class="container">
+    <div class="table-responsive">
+        <table class="table">
+            <tr>
+                <th class="project-reports-date-header">{{ UI_text.DATE_COLUMN_HEADER.value }}</th>
+                <th class="project-reports-project-header">{{ UI_text.PROJECT_COLUMN_HEADER.value }}</th>
+                <th class="project-reports-task-activity-header">{{ UI_text.TASK_ACTIVITY_HEADER.value }}</th>
+                <th class="project-reports-author-header">{{ UI_text.AUTHOR_COLUMN_HEADER.value }}</th>
+                <th class="project-reports-work-hours-header">{{ UI_text.WORK_HOURS_COLUMN_HEADER.value }}</th>
+                <th class="project-reports-description-header">{{ UI_text.DESCRIPTION_COLUMN_HEADER.value }}</th>
+                <th class="project-reports-creation-date-header">{{ UI_text.CREATION_DATE_COLUMN_HEADER.value }}</th>
+                <th class="project-reports-last-update-header">{{ UI_text.LAST_UPDATE_COLUMN_HEADER.value }}</th>
+                <th class="project-reports-editable-header">{{ UI_text.EDITED_COLUMN_HEADER.value }}</th>
+                <td class="project-reports-edit-button-header Invisible"></td>
+            </tr>
+            <tbody>
+                {% regroup object_list by date as reports_by_date %}
```

Comment:
Here you would use the `get_reports_ordered` method.

Reply:
Done